default.nix

```diff
@@ -200,7 +200,7 @@ in
       }
     ];
     network = {
-      sim2h_url = "ws://sim2h-test.holo.host:9003";
+      sim2h_url = "ws://localhost:9000";
```

Comment:
if these conductors are only used for our manual testing, seems like we should default to them being on a local network.